pylint/lint/pylinter.py

```diff
@@ -609,6 +621,38 @@ def register_checker(self, checker):
         if not getattr(checker, "enabled", True):
             self.disable(checker.name)
 
+    def enable_fail_on_messages(self):
+        vals = self.config.fail_on
+        if not vals:
+            return
+        all_cats = ["C", "R", "W", "E"]
```

Comment:
Isn't this a constant somewhere else ?

Reply:
Well that's embarrassing. I'm hoping that at the time I thought "later I should fix this to pull from wherever this is defined". Will fix.